### PR TITLE
Update release notes to reflect the upcoming support matrix changes

### PR DESCRIPTION
**Pull request by yuva29**
Signed-off-by: Yuva Shankar <yuva29@users.noreply.github.com>

### CODE REVIEW

**Comment by yuva29** [comment]
@rhim PTAL

File: websrc/source/documents/releasenotes/beta.md [hunk 1]
```diff
@@ -24,28 +24,42 @@ This release includes:
...
 
 Host OS
 
 - CentOS 7.x
+- RHEL 7.x (upcoming)
 
 Container Schedulers / Orchestrators
 
 - Kubernetes 1.4 
 - Docker 1.12 + Docker Swarm 1.2.5
```
Comment (rhim) [inline websrc/source/documents/releasenotes/beta.md]: 1.12.x

File: websrc/source/documents/releasenotes/beta.md [hunk 2]
```diff
@@ -24,28 +24,42 @@ This release includes:
...
+- Docker 1.13 (upcoming)
```
Comment (rhim) [inline websrc/source/documents/releasenotes/beta.md]: 1.13.x

File: websrc/source/documents/releasenotes/beta.md [hunk 3]
```diff
@@ -24,28 +24,42 @@ This release includes:
...
+	- L2/L3/ACI networks needs VMware port groups that need to be assigned with the following (For more information see [this](https://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vsphere.networking.doc%2FGUID-D5960C77-0D19-4669-A00C-B05D58A422F8.html)) :
+		- L2 or ACI - Port group needs to be configured using VLAN trunking to allow Virtual Guest Tagging.
+		- L3 - Port group can be configured using no VLAN if using External VLAN Tagging or assign a VLAN in the Port Group for Virtual Switch Tagging.
+
+- AWS (VXLAN only)
+
+Container Platforms
+
+- OpenShift Origin 1.4, Enterprise 3.4.x (upcoming)
+- DDC 2.0 (upcoming)
```
Comment (rhim) [inline websrc/source/documents/releasenotes/beta.md]: Change it to Docker Enterprise Edition, don't worry about version.

File: websrc/source/documents/releasenotes/beta.md [hunk 4]
```diff
@@ -24,28 +24,42 @@ This release includes:
...
 - Docker 1.12 + Docker Swarm 1.2.5
+- Docker 1.13 (upcoming)
 
 Infrastructure Support
 
 - Bare metal
-- vSphere
-- AWS
+- vCenter 
+	- L2/L3/ACI networks needs VMware port groups that need to be assigned with the following (For more information see [this](https://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vsphere.networking.doc%2FGUID-D5960C77-0D19-4669-A00C-B05D58A422F8.html)) :
```
Comment (rhim) [inline websrc/source/documents/releasenotes/beta.md]: Under vCenter, let's restructure as follows:
    - VM port group needs to be configured for correct networking mode (see [this](https://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vsphere.networking.doc%2FGUID-D5960C77-0D19-4669-A00C-B05D58A422F8.html) for more details). For L2 or ACI mode, port group needs to be configured using VLAN trunking to allow Virtual Guest Tagging. For L3 mode, port group can be configured using no VLAN if using External VLAN Tagging or assign a VLAN in the Port Group for Virtual Switch Tagging.

File: websrc/source/documents/releasenotes/beta.md [hunk 5]
```diff
@@ -24,28 +24,42 @@ This release includes:
...
+		- L2 or ACI - Port group needs to be configured using VLAN trunking to allow Virtual Guest Tagging.
+		- L3 - Port group can be configured using no VLAN if using External VLAN Tagging or assign a VLAN in the Port Group for Virtual Switch Tagging.
+
+- AWS (VXLAN only)
+
+Container Platforms
+
+- OpenShift Origin 1.4, Enterprise 3.4.x (upcoming)
```
Comment (rhim) [inline websrc/source/documents/releasenotes/beta.md]: Add as 
OpenShift Origin 1.4, OpenShift 3.4 (upcoming)

File: websrc/source/documents/releasenotes/beta.md [hunk 6]
```diff
@@ -24,28 +24,42 @@ This release includes:
...
-- vSphere
-- AWS
+- vCenter 
+	- L2/L3/ACI networks needs VMware port groups that need to be assigned with the following (For more information see [this](https://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vsphere.networking.doc%2FGUID-D5960C77-0D19-4669-A00C-B05D58A422F8.html)) :
+		- L2 or ACI - Port group needs to be configured using VLAN trunking to allow Virtual Guest Tagging.
+		- L3 - Port group can be configured using no VLAN if using External VLAN Tagging or assign a VLAN in the Port Group for Virtual Switch Tagging.
+
+- AWS (VXLAN only)
+
+Container Platforms
```
Comment (rhim) [inline websrc/source/documents/releasenotes/beta.md]: Integrated Container Platforms

File: websrc/source/documents/releasenotes/beta.md (hunk 2 above)
Comment (yuva29) [inline websrc/source/documents/releasenotes/beta.md]: Are we not supporting 1.13?

**Comment by yuva29** [comment]
@rhim [Preview.zip](https://github.com/contiv/contiv.github.io/files/860476/Archive.zip) contains the middleman output

@akamkar Please check if the changes are fine.

**Comment by yuva29** [comment]
@rhim Updated as requested [Preview.zip](https://github.com/contiv/contiv.github.io/files/866290/Preview.zip)

@akamkar PTAL

**Comment by rhim** [comment]
LGTM.

**Comment by akamkar** [comment]
LGTM

**Pull request merged**